unix/vncserver/vncserver.in

```diff
@@ -111,8 +119,13 @@ $default_opts{pn} = undef;
 # Load user-overrideable system defaults
 LoadConfig($vncSystemConfigDefaultsFile);
 
-# Then the user's settings
-LoadConfig($vncUserConfig);
+# Then the user's settings (location overrideable by previous system defaults)
+if (exists $config{'userconfig'}) {
+  LoadConfig($config{'userconfig'});
+  delete $config{'userconfig'};
```

Comment:
The original issue requested that the user config directory be overrideable in general, so I figured it may be worth providing this option similarly to the existing options for `rfbauth` and `.Xauthority`.

Reply:
It will now respect the XDG environment variables, which I could hope is enough.

I don't think most application allow more variability than that?

That also sounds like it is sufficient to resolve the original issue.

Reply:
A good amount of programs do support overriding config locations if they happen to only use single config files, but I suppose it isn't always necessary.

Reply:
Yeah, we have multiple files, so I think it's easy to make things confusing. Let's start with just the XDG stuff and we can see how the feedback is after that.